atomic_reactor/utils/operator.py

```diff
@@ -644,13 +676,21 @@ def set_related_images(self):
             old = by_name.setdefault(new.name, new)
             # Check for potential conflict (same name, different image)
             if new.image != old.image:
-                msg = ("{old.description}: {old.image} X {new.description}: {new.image}"
-                       .format(old=old, new=new))
-                conflicts.append(msg)
-
-        if conflicts:
-            raise RuntimeError("{} - Found conflicts when setting relatedImages:\n{}"
-                               .format(self.path, "\n".join(conflicts)))
+                description_conflict = conflicts_map.get(old, [])
+                description_conflict.append(new)
+                conflicts_map[old] = (description_conflict)
```

Comment:
The parenthesis were a carry over from an old implementation when I was storing a tuple of (old_description, [description_conflict]). I can remove that since I am now keying off the old description object to allow me to get the information from it directly.

Reply:
I am not sure how a defaultdict would save a line here. Don't we need to get the list out of the dictionary before we update it? We would need three lines regardless.

Reply:
```diff
- description_conflict = conflicts_map.get(old, [])
- description_conflict.append(new)
- conflicts_map[old] = (description_conflict)
+ conflicts_map.setdefault(old, []).append(new)
```
This is how you could save some lines. Optional for me

Reply:
That is awesome. I used the other construct because that is what I determined was the best when struggling with it previously... This is definitely going in my back pocket.

```
tests/utils/test_operator.py ........................................... [ 31%]
........................................................................ [ 61%]
........................................................................ [ 92%]
..................                                                       [100%]
```

Unit tests tell me that it is all good!